plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -164,6 +197,21 @@ export class StorageTaskBroker implements TaskBroker {
   }
 
   private deferredDispatch = defer();
+  private taskManagerRegistry = new Map<String, TaskManager>();
```

Comment:
This assumes that all task managers are running in the same scaffolder instance right? That's not always the case. Like other backend plugins the scaffolder needs to be horizontally scalable to this communication needs to happen via the database or some other common point, regardless of what scaffolder instance the abort request hits in relation to the task manager.

Reply:
Ok I see, have to think about this case.
How's that possible in practice to launch multiple instances of the same plugin? Do you mean the case when to run multiple instances of the backstage in K8s cluster which points to the same database or there are some other options to achieve it?

Reply:
Yep, exactly that, multiple instances that share a DB

Reply:
The concern is addressed